src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+
+.body {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &-image {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: $text-accent-color;
+
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: $main-color;
+}
+
+.nav__link.is-active {
+  color: $main-color;
+}
+
+.nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  height: 4px;
+  border-radius: 2px;
+  background-color: $main-color;
+}
+
+.cards {
+  grid-template-columns: repeat(1, 200px);
+
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+  width: 1024px;
```

Comment:
The `.cards` class is using `grid-template-columns` within a flex container. This might not work as intended because `grid-template-columns` is a grid property, not a flexbox property. Consider using `flex-basis` or `flex-grow` for flex containers.